editor/scene/particles_editor_plugin.cpp

```diff
@@ -84,17 +96,64 @@ void ParticlesEditorPlugin::_menu_callback(int p_idx) {
 		} break;
 
 		case MENU_RESTART: {
-			edited_node->call("restart");
+			if (EditorNode::get_singleton()->get_editor_selection()->get_selected_nodes().size() == 1) {
+				edited_node->call("restart");
+			} else {
+				for (int i = 0; i < mne->get_node_count(); i++) {
```

Comment:
It would be more efficient to introduce `LocalVector<Object> edited_nodes` and use it instead of `edited_node` when multiple nodes are selected.